### ISSUE TITLE
Publish "Mocks Server Administration API Client"

### ISSUE THREAD

**Issue by javierbrea** [mocks-server/admin-api-client#1]
It is desirable to publish the first version of the @mocks-server/admin-api-client package, which will allow users to interact programmatically with the Mocks Server administration REST API.

In the first version It should expose methods to perform some basic operations such as:
- Change delay time
- Change current behavior

Here goes a proposal for the interface:

```js
const AdminApiClient = require("@mocks-server/admin-api-client");

const apiClient = new AdminApiClient("http://localhost:3100");

apiClient.setBehavior("foo-behavior").then(() => {
  console.log("Behavior changed");
});

apiClient.setDelay(3000).then(() => {
  console.log("Delay changed");
});

```

**Comment by javierbrea** [comment]
Dependent of mocks-server/main#20

**Comment by javierbrea** [comment]
First release merged in PR #2 , and published.
